src/AzureClient/Visualization/ResourceEstimationEncoder.cs

```diff
@@ -92,27 +97,50 @@ public record class ResourceEstimationToHtmlEncoder(ILogger? Logger = null) : IR
                         }
 
                         .aqre-tooltip:hover .aqre-tooltiptext {
-                            visibility: visible;
+                          visibility: visible;
```

Comment:
I have an impression that this indentation change was not planned (based on indentation at green L85-).

Reply:
Fixed.